app/jobs/regular/user_email.rb

```diff
@@ -22,6 +22,15 @@ def execute(args)
       # of extra work when emails are disabled.
       return if quit_email_early?
 
+      send_user_email(args)
+
+      if args[:user_id].present? && args[:type] == :digest
+        # Record every attempt at sending a digest email, even if it was skipped
+        UserStat.where(user_id: args[:user_id]).update_all(digest_attempted_at: Time.zone.now)
```

Comment:
TIL/FYI - a shorter alternative:

```suggestion
        UserStat.where(user_id: args[:user_id]).touch_all(:digest_attempted_at)
```

Reply:
Looks like this will also change the `updated_at` column... which I don't think we want. I think maybe `update_all` is a little more explicit/obvious? (Confirmed by the fact that `touch_all` is a "TIL" for both of us? ;) )

Reply:
It would bump `updated_at` column… if `UserStat` had one! :trollface: